pkg/server/apiinternal/api_internal_authentication.go

```diff
@@ -0,0 +1,22 @@
+// Copyright 2025 The Cockroach Authors.
+//
+// Use of this software is governed by the CockroachDB Software License
+// included in the /LICENSE file.
+
+package apiinternal
+
+// registerAuthenticationRoutes sets up the authentication REST endpoints.
+func (r *apiInternalServer) registerAuthenticationRoutes() {
+	routes := []route{
+		// Login endpoint
+		{POST, "/login", createHandler(r.login.UserLogin)},
```

Comment:
Should we consider using the constants authserver.LoginPath and authserver.LogoutPath instead of the hardcoding here?